TempleDAO/temple
# feat: support safe provider and web app

# Description

- Adds Safe connector to wagmi
- Bypass signature requirement if using Safe

Resolves https://github.com/TempleDAO/temple/issues/759

To test in safe app:
- Go to https://app.safe.global
- Click Apps
<img width="224" alt="Screenshot 2023-02-21 at 1 08 25 PM" src="https://user-images.githubusercontent.com/99344331/220425493-e4a3726d-c90b-4f15-b6b7-bbf7aff28fc1.png">
- My custom apps -> Add custom app

<img width="707" alt="Screenshot 2023-02-21 at 1 08 30 PM" src="https://user-images.githubusercontent.com/99344331/220425522-cf0fab80-da4d-471b-9b76-425fb18b02ba.png">

- Enter https://pr-769-preview.templedao.link/
- Then when the app is loaded, connect your wallet and choose Safe

TODO: 
- Should we hide safe when not loaded in safe app?



# Checklist
- [ ] This PR is targeting the correct branch
